src/Troubleshooting/index.md

```diff
@@ -11,3 +11,76 @@ This is due to a Roblox update causing an incompatibility with Wayland's handlin
 This specific problem can be the same as many other root causes, a common one is your graphics card not supporting Vulkan, or is missing the graphics libraries required for Wine; you can see [Lutris Docs: Installing Drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md)
 
 This may also be your GPU not supporting Vulkan or not supporting modern Vulkan; Use the OpenGL renderer, or set the installed DXVK version to one which includes a legacy version of Vulkan that your GPU supports.
+
+#### "Make sure there are no stable wineservers running"
+
+This is because it failed to run the ESync Shared Memory File, so you will manually have to kill the `wineserver`.
+
+You need to end the `wineserver` process. Using System Monitor from many Desktop Environments, or TUI System Monitors like Btop.
+Or you can wait until the wineserver process will end.
+
+If you want to make your life easier, you can also type `vinegar kill` in the terminal.
+
+#### Roblox Studio stuck on Loading Screen
+
+You need to set the DPI to 97. By executing the command `vinegar exec winecfg`, then open the Graphics section and you can change the DPI to 97.
+
+The only downside of this is that the fonts for Roblox will be a bit blurry, an even number for the DPI would fix this issue, but Roblox will have a bigger interface on both Player and Studio.
+
+#### WebView2 installer stuck on installing
+
+Theres an unknown reason for how this error occured, so unfortunately we can't explain whats the issue. Luckily we do have the solution.
+
+You must change the Windows Version to 7,
+You can set the Windows Version to 7 by executing the command `vinegar exec winecfg /v win7`. And WebView2 will successfully install.
+
+#### Cannot login to Studio or Player due to Verification System not working
+
+This is because of the horrible way WebView2 runs on Wine, and it would just show a White Screen when the Verification System is trying to load.
+
+Login to the [Roblox Website](https://roblox.com) and join a game from there, The login will be synced perfectly with Studio and Player.
+
+#### "Your GPU is incompatible" error when launching Roblox Studio
+
+Disable DXVK in the configuration file of Vinegar, Downside of this is that there will be less optimizations due to no Vulkan Optimizations.
+[Vinegar Docs: Setting Configuration File](https://vinegarhq.github.io/Configuration/index.html)
```

Comment:
One of the most common causes for this issue is the user not having a childwindow patch applied in their wine build. 

We should probably point this out